File: Arrays/Minimum Swaps 2/minimumswaps.py
# Complete the minimumSwaps function below.
def minimumSwaps(arr):
    res = 0

    for i in range(0, len(arr) - 1):
        while arr[i] != i + 1:
            tmp = arr[arr[i] - 1]
            arr[arr[i] - 1] = arr[i]
            arr[i] = tmp
            res += 1

    return res

File: Arrays/Minimum Swaps 2/minimumswaps.cpp
// Complete the minimumSwaps function below.
int minimumSwaps(vector<int> arr) {
   int i,c=0,n=arr.size();
    for(i=0;i<n;i++)
    {
        if(arr[i]==(i+1))
            continue;

        swap(arr[i],arr[arr[i]-1]);
        c++;
        i--;
    }
    return c;
}
